std/encoding/binary_test.ts

```diff
@@ -160,5 +160,3 @@ Deno.test(async function testWriteVarnumLittleEndian(): Promise<void> {
   await buff.read(data);
   assertEquals(data, new Uint8Array([0x01, 0x02, 0x03, 0x04]));
 });
-
-Deno.runTests();
```

Comment:
This is very suspicious, might have been that some of the tests were run twice... We need to guard against multiple calls to `runTests`